configure.ac

```diff
@@ -852,16 +852,16 @@ if test "$want_dag" != no; then
 		# included if there's a found-action (arg 3).
 		saved_ldflags=$LDFLAGS
 		LDFLAGS="-L$dag_lib_dir"
-		AC_CHECK_LIB([dag], [dag_attach_stream], [dag_streams="1"], [dag_streams="0"])
```

Comment:
Removing this means that the configure script won't fail if the library doesn't have `dag_attach_stream()`.  I added a change in 283a5c415ea214fe4b2b53a29a620f73ad949057 to do so.